stitch/options.py

```diff
@@ -44,3 +47,29 @@ def validate(self, obj, value):
         if isinstance(value, str) or value is None:
             return value
         self.error(obj, value)
+
+
+class LangMapper:
+    def __init__(self, meta):
+        self._kernels = self._read_dict(meta, KERNELS)
+        self._styles = self._read_dict(meta, STYLES)
+
+    @staticmethod
+    def _read_dict(meta, dict_name):
+        if dict_name in meta:
+            try:
+                ret = {key: val['c'][0]['c'] for key, val in meta[dict_name]['c'].items()}
+                if all([isinstance(key, str) and isinstance(val, str) for key, val in ret.items()]):
+                    return ret
+            except (KeyError, IndexError, AttributeError):
+                pass
+            print('WARNING: Invalid {0} metadata section was ignored.'.format(dict_name))
```

Comment:
This should either be a `warnings.warn` or raise an exception. I think I'd prefer to raise a `ValueError`.

Reply:
Guess so. It would be easier to debug with `ValueError`.

Reply:
And no, I cannot think of any edge cases around `LangMapper` - it would raise `ValueError` in all that cases.